ns1/__init__.py

```diff
@@ -371,7 +377,8 @@ def createZone(
         If zoneFile is specified, upload the specific zone definition file
         to populate the zone with.
 
-        :param str zone: zone name, like 'example.com'
+        :param str zone: zone FQDN, like 'example.com'
+        :param str zone: zone name override, name will be zone FQDN if omitted
```

Comment:
```suggestion
        :param str name: zone name override, name will be zone FQDN if omitted
```

It is possible in Python to declare keyword-only arguments:

https://peps.python.org/pep-3102/

But we don't have that now, so order matters. 

If you want to reorder the parameters like this you'll need to make a major release, as it can break the API. Alternately you can put it at the end. :smile:

Reply:
We're < 1.0 so we don't have to do a major release for such things (I think). But we will be breaking stuff!